chompfile.toml

```diff
@@ -14,9 +14,9 @@ run = 'cargo build --release --locked'
 
 [[task]]
 name = 'test'
-dep = 'build'
+deps = ['build', 'install']
 cwd = 'test'
-run = '../target/debug/chomp test'
+run = 'chomp test'
```

Comment:
We need to ensure that the tests run against the local Chomp version though.

Reply:
After install task, wouldn't be the global chomp the new debugging build? So it's always updated and it's the `target/debug/chomp`!

Let me know if I'm wrong.